api/v1alpha3/virtualmachine_types.go

```diff
@@ -357,6 +361,65 @@ type VirtualMachineCdromSpec struct {
 	AllowGuestControl *bool `json:"allowGuestControl,omitempty"`
 }
 
+// VirtualMachineCryptoSpec defines the desired state of a VirtualMachine's
+// encryption state.
+type VirtualMachineCryptoSpec struct {
+	// +optional
+
+	// ClassName describes the name of the EncryptionClass resource
+	// used to encrypt this VM.
+	//
+	// Please note, this field is not required to encrypt the VM. If the
+	// underlying platform has a default key provider, the VM may still be fully
+	// or partially encrypted depending on the specified storage and VM classes.
+	//
+	// If there is a default key provider and an encryption storage class is
```

Comment:
Hmm.  I think this has to be more nuanced than that.

I think by "VM's home files", you mean files in VM_HOME directory.  But that also include PVC disks (FCDs can be created in either the designated fcd/ directory, or VM_HOME).

Reply:
No, by VM home files I mean any files in the VM directory that are *not* disks.